### DISCUSSION TITLE
Ajouter une carte "vous êtes libéré de prison"

### DISCUSSION THREAD

**Discussion by BastLast** [Crownicles/Crownicles#1180] [Nouvelles idées]
Prend la forme dun mini event. Très rare (1)

La personne obtient une carte carte "vous êtes libéré de prison"

Il ne peut y avoir qu'une seule carte vous etes libéré de prison en circulation à la fois (en gros si un autre joueur en a déjà une ca affiche le pseudo du joueur qui la possède)

Si le joueur qui la possède déjà n'a pas joué depuis plus de 72 heures, la personne qui a le mini event peut choisir de voler la carte. (et du coup gagne la carte) il y a 5% de chance d'aller en prison 

Le détenteur de la carte est marqué d'un emoji sur son profil et dans le top

La personne qui est en prison avec une carte libéré de prison peut etre libérée gratuitement avec la commande unlock ( mais il faut quand même que quelqu'un d'autre la libère)

**Comment by Kyusaor** [comment]
Je pense que ça fait assez peu 1 pour la totalité des joueurs. Suffit qu'il y ait un joueur assez actif qui ne va pas en taule au premier mini event prison ça peut durer relativement longtemps

En mettre genre 3 ou 4 en circulation me paraît mieux

**Reply by BastLast** [reply]
En fait je pense qu'une autre solution serait d'offrir la possibilité de la voler tout le temps, mais que la probabilité de réussite de vol augmente en fonction du temps inactif.

A voir ce qu'il se passe en cas d'échec du vol ( pas qqch de positif lol)

**Reply by OscarT54** [reply]
Baser la probabilité sur l'activité poserai le même problème. Il suffirais qu'elle tombe entre les mains d'un joueur même moyennement actif (1 !r/8 heures) et elle deviens quasi-inobtenable. Si cette solution est mise en place il faudrais en effet plusieurs cartes (genre 5). Sinon on pourrais baser la probabilité sur la durée depuis laquelle le joueur a la carte : quasiment involable si il viens de l'obtenir et sûr de l'avoir si il l'a depuis 1/2 mois

**Reply by BastLast** [reply]
Ok je pense que le mieux est d'avoir quelques cartes genre 3 ou 4  et d'équilibrer pour qu'elles puissent être conservées quelques semaines.

En gros les pénalités en cas d'échec du vol seront assez grosses

Je propose une amende aléatoire entre 500 et 5000 et si le joueur a pas assez, il va en prison ( et l'amende est annulée )

Si le joueur fait le choix de ne pas voler la carte il a 10 % de chance de recevoir une petite récompense genre 100 pièces
